src/IO/H5/ReadTabulated.hpp

```diff
@@ -0,0 +1,55 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class h5::Tabulated
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <cstdint>
+#include <hdf5.h>
+#include <string>
+#include <vector>
+
+#include "IO/H5/Object.hpp"
+#include "IO/H5/OpenGroup.hpp"
+
+/// \cond
+class Matrix;
+/// \endcond
+
+namespace h5 {
+/*!
+ * \ingroup HDF5Group
+ *
+ * Contains functions to read in rank-1 and rank-3 datasets from an h5 file.
+ */
+class Tabulated : public h5::Object {
```

Comment:
I expected the declaration of the class Tabulated to be in Tabulated.hpp - is there a reason why it's done this way?